trendyqc/trend_monitoring/backend_utils/plot.py

```
@@ -300,32 +302,17 @@ def format_data_for_plotly_js(plot_data: pd.DataFrame) -> tuple:
-            "#ff65ff",  # fushia
-            "#6600cc",  # purple
-            "#1c6dff",  # blue
-            "#6ddfff",  # light blue
-        ],
-    }
+    try:
+        colors = settings.PLOTTING_COLORS
+    except AttributeError:
+        raise ImproperlyConfigured(
+            "PLOTTING_COLORS setting is missing. Please define it in your settings file."
+        )
```

Ruff (0.8.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)